pkg/kubectl/describe.go

```diff
@@ -416,6 +416,9 @@ func describePod(pod *api.Pod, rcs []api.ReplicationController, events *api.Even
 		fmt.Fprintf(out, "Namespace:\t%s\n", pod.Namespace)
 		fmt.Fprintf(out, "Image(s):\t%s\n", makeImageList(&pod.Spec))
 		fmt.Fprintf(out, "Node:\t%s\n", pod.Spec.NodeName+"/"+pod.Status.HostIP)
+		if pod.Status.StartTime != nil {
+			fmt.Fprintf(out, "Start Time:\t%s\n", pod.Status.StartTime.Time.Format(time.RFC1123Z))
```

Comment:
In  making things human-readable, I think it'd be worth having both at the very least (or just keeping as the original string).

In GitHub's UI, I find it really annoying when I see some days that are like "July something" and some that are like "27 days ago". Maybe subtracting 27 days is easy for most people, but it's hard for me (if you're trying to diff times to, say, the right hour.)

So you could do `Tue, 14 Jul 2015 17:42:40 -0700 (31d ago)`

but there might actually be some value in just keeping things in standardized formats for parsing with other tools